src/pages/HomePage.js

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/macro'
+import Alert from '../components/Alert/Alert'
+import FilterForm from '../components/FilterForm/FilterForm'
+import Recipe from '../components/Recipe/Recipe'
+import SearchBar from '../components/SearchBar/SearchBar'
+
+export default function HomePage({
+  onRecipeSearch,
+  text,
+  dietLabels,
+  allergiesLabels,
+  cuisineTypes,
+  onFindClicked,
+  recipes,
+}) {
+  return (
+    <>
+      <PageLayout>
+        <SearchBar onRecipeSearch={onRecipeSearch} />
+        <Alert text={text} />
+        <FilterForm
+          dietLabels={dietLabels}
+          allergiesLabels={allergiesLabels}
+          cuisineTypes={cuisineTypes}
+          onFindClicked={onFindClicked}
+        />
+
+        {recipes.map(recipe => (
```

Comment:
you could also use destructuring assignment :) like `{recipes.map({id, recipe} => `

Reply:
Thank you for your suggestion! I'll change it.